### PR TITLE
Set topology view as landing page of dev-console

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Task : https://jira.coreos.com/browse/ODC-494

devconsole-landing-page

### CODE REVIEW

File: frontend/public/extend/devconsole/shared/components/PerspectiveSwitcher.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const PerspectiveSwitcher: React.FunctionComponent<Props> = (props: Props
     }
     return (
       <MegaMenuItem
-        to="/dev"
+        to="/"
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/shared/components/PerspectiveSwitcher.tsx]: This should still point to `/dev`. If the user right clicks and chooses to open in a new tab we'd end up opening up the admin perspective.

File: frontend/public/components/app.jsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ const DefaultPage = connect(mapPerspectiveStateToProps)(
     if (openshiftFlag) {
       // TODO - We should be using the link utility to create these links with perspective.
       return lastViewedPerspective && lastViewedPerspective !== 'admin' ? (
-        <Redirect to={`/${lastViewedPerspective}`} />
+        <Redirect to={`/${lastViewedPerspective}/topology`} />
```
Comment (christianvogt) [inline frontend/public/components/app.jsx]: You need to add a new route for `/dev` that redirects to the default page. You can use react-router's `Redirect` component.
Keep this as is.

**Review by christianvogt — CHANGES_REQUESTED** [review]

File: frontend/public/components/app.jsx (hunk 2 above)
Comment (rohitkrai03) [reply frontend/public/components/app.jsx]: Yes, changes to this `Redirect` will affect other perspectives as well. Right now its generic enough for all perspective.

**Review by rohitkrai03 — APPROVED** [review]
LGTM

**Review by christianvogt — APPROVED** [review]

**Pull request merged**